<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>649</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>686</width>
    <height>10</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1800</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>761</width>
      <height>571</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="aggiungifigura">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(245,245,245);
border: 1px solid gray;
color:#333;
}</string>
    </property>
    <property name="text">
     <string>Aggiungi Figura</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rimuoviTuttoButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>261</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(245,245,245);
border: 1px solid gray;
color:#333;
}</string>
    </property>
    <property name="text">
     <string>Rimuovi Tutto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rimuoviVerticeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>100</y>
      <width>210</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(180,180,180);
border: 1px solid gray;
color:white;
}

QPushButton:pressed{
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
																stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>Rimuovi Vertice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aggiungiVerticeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>40</y>
      <width>210</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(180,180,180);
border: 1px solid gray;
color:white;
}

QPushButton:pressed{
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
																stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>Aggiungi Vertice</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Strumenti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>560</y>
      <width>221</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(245,245,245);
border: 1px solid gray;
color:#333;
}</string>
    </property>
    <property name="text">
     <string>Chiudi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoommeno">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>450</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(245,245,245);
border: 1px solid gray;
color:#333;
}</string>
    </property>
    <property name="text">
     <string>Zoom 
-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoompiu">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(245,245,245);
border: 1px solid gray;
color:#333;
}</string>
    </property>
    <property name="text">
     <string>Zoom 
+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rimuoviFiguraButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>160</y>
      <width>211</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(180,180,180);
border: 1px solid gray;
color:white;
}

QPushButton:pressed{
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
																stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>Rimuovi Figura</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="PropertyTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>261</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sottraiFiguraButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>280</y>
      <width>211</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(180,180,180);
border: 1px solid gray;
color:white;
}

QPushButton:pressed{
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
																stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>Sottrai Figura</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sommaFiguraButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>220</y>
      <width>210</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(180,180,180);
border: 1px solid gray;
color:white;
}

QPushButton:pressed{
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
																stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>Somma Figura</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
